modeling/07_Landsat_9_GTB.Rmd

```diff
@@ -0,0 +1,551 @@
+    crsTransform = img_crsTrans
+  )
+
+  missDate = image.get('missDate')
+
+  # Create a feature with the calculated area and properties
+  a = area.first().set({'missDate': missDate, 
+                        'area_ha': aoi_area_ha, 
+                        'extent': 'aoi'})
+  
+  return ee.FeatureCollection(a)
+
+
+def calcArea_nosc(image):
+  areaImage =  (image
+    .multiply(ee.Image.pixelArea()).divide(1e5)
+    .clip(aoi_no_sc_ee.geometry()))
+  
+  area = areaImage.reduceRegions(
+    collection = aoi_no_sc_ee,
+    reducer = ee.Reducer.sum().forEachBand(areaImage),
+    crs = img_crs,
+    crsTransform = img_crsTrans
+  )
+  
+  missDate = image.get('missDate')
+  
+  # Create a feature with the calculated area and properties
+  a = area.first().set({'missDate': missDate, 
+                        'area_ha': aoi_no_sc_area_ha, 
+                        'extent': 'aoi no sc'})
+  
+  return ee.FeatureCollection(a)
+
+
+def calcArea_sc(image):
+  areaImage =  (image
+    .multiply(ee.Image.pixelArea()).divide(1e5)
+    .clip(aoi_sc_ee.geometry()))
+
+  area = areaImage.reduceRegions(
+    collection = aoi_sc_ee,
+    reducer = ee.Reducer.sum().forEachBand(areaImage),
+    crs = img_crs,
+    crsTransform = img_crsTrans
+  )
+
+  missDate = image.get('missDate')
+
+  # Create a feature with the calculated area and properties
+  a = area.first().set({'missDate': missDate, 
+                        'area_ha': aoi_sc_area_ha, 
+                        'extent': 'shoreline contamination'})
+  
+  return ee.FeatureCollection(a)
+
+```
+
+And then map them over all the stacks
+
+```{python}
+
+allAreas_l9_aoi = lightStack_l9.map(calcArea_aoi)
+
+allAreas_l9_nosc = lightStack_l9.map(calcArea_nosc)
+
+allAreas_l9_sc = lightStack_l9.map(calcArea_sc)
+
+```
+
+
+#### Landsat 9
+
+```{python}
+## export to drive
+export_summary_5_aoi = (ee.batch.Export.table.toDrive(
+  collection = allAreas_l9_aoi,
+  selectors = ['missDate', 'classified', 'cloud', 'openWater', 'lightNSSed','OSSed', 'dNSSed', 'area_ha', 'extent'],
+  description = 'gradientTreeBoost_landsat_9_stack_v' + v_date,
+  folder = 'eePlumB_classification_v2024-01-08',
+  fileFormat = 'csv'))
+
+export_summary_5_aoi.start()
+
+export_summary_5_nosc = (ee.batch.Export.table.toDrive(
+  collection = allAreas_l9_nosc,
+  selectors = ['missDate', 'classified', 'cloud', 'openWater', 'lightNSSed','OSSed', 'dNSSed', 'area_ha', 'extent'],
+  description = 'gradientTreeBoost_landsat_9_stack_nosc_v' + v_date,
+  folder = 'eePlumB_classification_v2024-01-08',
+  fileFormat = 'csv'))
+
+export_summary_5_nosc.start()
+
+
+export_summary_5_sc = (ee.batch.Export.table.toDrive(
+  collection = allAreas_l9_sc,
+  selectors = ['missDate', 'classified', 'cloud', 'openWater', 'lightNSSed','OSSed', 'dNSSed', 'area_ha', 'extent'],
+  description = 'gradientTreeBoost_landsat_9_stack_sc_v' + v_date,
+  folder = 'eePlumB_classification_v2024-01-08',
+  fileFormat = 'csv'))
+
+export_summary_5_sc.start()
```

Comment:
This will not affect any of the outcomes from running the code but for consistency with your other code, I think all of these objects should start with `export_summary_9_`.